### PR TITLE
feat(kmesh-cp) add kubernetes tags automatically

### LINKED ISSUE #3367: Auto add labels for useful pod info
### Description

Some examples:

```
k8s.kuma.io/namespace
```

This makes it simpler to create TrafficPermission for an entire namespace for example.

### PR DESCRIPTION
**Pull request by jakubdyszkiewicz**
### Summary

Add `k8s.kuma.io/namespace` tag automatically so we can build policy based on it.

### Issues resolved

Fix #3367

### Documentation

- [X] https://github.com/kumahq/kuma-website/pull/623

### Testing

- [X] Unit tests
- [X] E2E tests
- [X] No testing on Universal
- [X] Manual testing on Kubernetes

### Backwards compatibility

- [X] No changes in UPGRADE.md
- [X] Add `backport-to-stable` label if the code is backwards compatible. Otherwise, list breaking changes.

### CODE REVIEW

File: pkg/plugins/runtime/k8s/controllers/inbound_converter.go [hunk 1]
```diff
@@ -12,6 +12,8 @@ import (
 	util_k8s "github.com/kumahq/kuma/pkg/plugins/runtime/k8s/util"
 )
 
+const NamespaceTag = "k8s.kuma.io/namespace"
```
[the remark this answers is not in the document]
Comment (lahabana) [reply pkg/plugins/runtime/k8s/controllers/inbound_converter.go:15]: And maybe `k8s.kuma.io/service` which is the serviceName in k8s. That way you have the info in `kuma.io/service` in separate tags.
Usage of `k8s.kuma.io/service` would be for example adding policy to services exposed on all pods (.e.g metrics)

File: pkg/plugins/runtime/k8s/controllers/pod_converter_test.go [hunk 2]
```diff
@@ -378,7 +375,8 @@ var _ = Describe("InboundTagsForService(..)", func() {
 			// given
 			pod := &kube_core.Pod{
 				ObjectMeta: kube_meta.ObjectMeta{
-					Labels: given.podLabels,
+					Namespace: "test",
```
Comment (lahabana) [inline pkg/plugins/runtime/k8s/controllers/pod_converter_test.go:378]: Would this test be less confusing in the namespace of the service and the namespace of pods would match?

**Review by lahabana — APPROVED** [review]
Nice! I have a couple of comments but this change LGTM!

File: pkg/plugins/runtime/k8s/controllers/inbound_converter.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (lahabana) [reply pkg/plugins/runtime/k8s/controllers/inbound_converter.go:15]: @jakubdyszkiewicz said:

>  I think that k8s.kuma.io/port is a bit generic. It always raises the question of “but what port?“. Should this be k8s.kuma.io/service-port?

I don't really mind either way so if you think service-port is better go for it!

[the remark this answers is not in the document]
Comment (jakubdyszkiewicz) [reply pkg/plugins/runtime/k8s/controllers/inbound_converter.go]: added. I also put `k8s.kuma.io/service-name`. Although `-name` suffix sounds redundant, I find `k8s.kuma.io/service` to be too close to `kuma.io/service`

**Review by lahabana — APPROVED** [review]

**Pull request merged**